src/mobu/factory.py

```diff
@@ -37,9 +37,11 @@ class ProcessContext:
 
     def __init__(self, http_client: AsyncClient) -> None:
         self.http_client = http_client
-        logger = structlog.get_logger("mobu")
-        gafaelfawr = GafaelfawrStorage(http_client, logger)
-        self.manager = FlockManager(gafaelfawr, http_client, logger)
+        self.logger = structlog.get_logger("mobu")
+        self.gafaelfawr = GafaelfawrStorage(self.http_client, self.logger)
```

Comment:
Do we actually reference logger and gafaelfawr from elsewhere?  If I had to guess, this is another of the things Russ is pretty hard-nosed about, that you don't expose symbols that you don't mean to be exported for external use.  (Even if a leading `_` is basically advisory anyway.)